db/data/20240617164456_update_template_version_permit_types.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+class UpdateTemplateVersionPermitTypes < ActiveRecord::Migration[7.1]
+  def up
+    PermitType.find_by_code("low_residential").update(
+      name: "1-4 Unit detached housing",
+      description: "1-4 units: Detatched dwellings, duplexes",
+    )
+    PermitType.find_by_code("medium_residential").update(
+      name: "4+ Unit housing",
+      description: "Part 9 townhouses, small apartment buildings",
+    )
+    PermitType.find_by_code("high_residential").update(
+      name: "High density appartment buildings",
+      description: "Highest density residential structures",
+    )
+
+    update_template_version_jsons
+  end
+
+  def down
+    PermitType.find_by_code("low_residential").update(
+      name: "Low desnity residential",
+      description: "1-4 units: Detatched dwellings, duplexes",
+    )
+    PermitType.find_by_code("medium_residential").update(
+      name: "Medium density Residential",
+      description: "Part 9 townhouses, small apartment buildings",
+    )
+    PermitType.find_by_code("high_residential").update(
+      name: "High density residential",
+      description: "Highest density residential structures",
+    )
+
+    update_template_version_jsons
```

Comment:
I think the down is mistakenly copied over from up. keep it blank is there is no need for down